### DISCUSSION TITLE
Manuel en Français

### DISCUSSION THREAD

**Discussion by 5axes** [supermerill/SuperSlicer#1372] [General]
Bonjour,

J'ai mis en ligne la version Md (Markdown) du manuel (non officiel) : 
https://github.com/5axes/SuperSlicer-FRDocumentation/blob/main/src/superslicer.md

Cela permettra d'être plus 'collaboratif' dans la rédaction... un prochain travail de révision sera à faire pour je l'espère compléter les textes.

**Comment by 5axes** [comment]
Pour le suivi des modifications j'ai mis à jour le document partagé GoogleSheet fait au départ pour la version offcielle : 
[https://docs.google.com/spreadsheets/d/1OUTuHtJ8CQbieKgvZqYqVsYos-c-FmGSYwkhxql8Ddg](https://docs.google.com/spreadsheets/d/1OUTuHtJ8CQbieKgvZqYqVsYos-c-FmGSYwkhxql8Ddg)
Avec maintenant un onglet  **workplan_fr**

**Reply by 5axes** [reply]
onglet **Variable_list_fr** pour la partie variable en français .. peut être un jour car pour le moment ca représenterait 500 Variables

**Reply by 5axes** [reply]
Pour les variables :  523/153

**Reply by supermerill** [reply]
la plupart des variables sont définis dans printconfig.cpp, si besoin.

**Reply by 5axes** [reply]
304/539  56% on progresse.

**Reply by 5axes** [reply]
449/561  80.04% encore une vingtaine de trouvé à mettre à jour dans https://github.com/5axes/SuperSlicer-FRDocumentation/blob/main/src/macros/variable_list.md .

**Comment by 5axes** [comment]
Bon les pages sont à peu près toutes créées .. On repart pour un tour de révision:
Variables : 572 | 565
https://docs.google.com/spreadsheets/d/1OUTuHtJ8CQbieKgvZqYqVsYos-c-FmGSYwkhxql8Ddg

**Comment by 5axes** [comment]
Milling_cutter : https://github.com/5axes/SuperSlicer-FRDocumentation/blob/main/src/variable/milling_cutter.md

Pas d'infos sur ce paramètre et pas d'exemple .. Si quelqu'un a plus de renseignement ?

**Reply by supermerill** [reply]
C'est le nombre de fraiseuses (mais en fait, il ne sert à rien d'en mettre plus qu'une, donc c'est souvent 0 ou 1)
Ca rajoute un onglet avec les propriété de cet outil (un peu comme avec les extrudeurs)
C'est utilisé dans output-> milling post-process
